superset-frontend/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -145,7 +148,14 @@ class HeaderActionsDropdown extends React.PureComponent {
         this.props.showPropertiesModal();
         break;
       case MENU_KEYS.DOWNLOAD_AS_IMAGE:
-        downloadAsImage('.dashboard', this.props.dashboardTitle)(domEvent);
+        setTimeout(
+          () =>
+            downloadAsImage(
+              screenshotNodeSelector,
+              this.props.dashboardTitle,
+            )({ currentTarget }),
+          1000,
```

Comment:
If this is the solution that ends up being needed, can you:
1. Make the timeout a descriptively named constant
2. Add comments explaining why this is necessary

Reply:
Done